sources/packages/forms/src/form-definitions/programinformationrequest.json

```diff
@@ -2634,6 +2665,65 @@
       "inputType": "hidden",
       "id": "e6z1qd9",
       "addons": []
+    },
+    {
+      "input": true,
+      "tableView": false,
+      "key": "selectedOfferingEndDate",
+      "label": "Selected offering end date",
+      "protected": false,
+      "unique": false,
+      "persistent": true,
+      "type": "hidden",
+      "tags": [],
+      "conditional": {
+        "show": "",
+        "when": null,
+        "eq": ""
+      },
+      "properties": {},
+      "lockKey": true
+    },
+    {
+      "input": true,
+      "tableView": false,
+      "key": "maximumAllowedDaysAfterStudyEndDate",
+      "label": "Maximum allowed days after study end date",
+      "protected": false,
+      "unique": false,
+      "persistent": true,
+      "type": "hidden",
+      "tags": [],
+      "conditional": {
+        "show": "",
+        "when": null,
+        "eq": ""
+      },
+      "properties": {},
+      "calculateValue": "/**\n * Maximum number of days past the study period end date\n * beyond when an application will require StudentAid BC\n * approval for funding.\n */\n \nvalue = 42;",
+      "calculateServer": true,
+      "lockKey": true
+    },
+    {
+      "input": true,
+      "tableView": false,
+      "key": "selectedStudyEndDatePastAllowedDays",
+      "label": "Selected study end date past maximum allowed days",
+      "protected": false,
+      "unique": false,
+      "persistent": true,
+      "type": "hidden",
+      "tags": [],
+      "conditional": {
+        "show": "",
+        "when": null,
+        "eq": ""
+      },
+      "properties": {},
+      "calculateValue": "/**\n * Validates if the date provided is within given days\n * before now.\n * e.g. If the current date is 10-JAN-2023 and\n * given number of days is 4, it validates if the given date\n * is on or after 06-JAN-2023.\n * \n */\n \nfunction isAfterGivenDaysBeforeNow(\n  givenDate,\n  numberOfDaysBeforeToday\n) {\n  return moment\n    .utc(givenDate)\n    .isAfter(\n      moment\n        .utc(moment.utc(new Date()).format(\"YYYY-MM-DD\"))\n        .subtract(numberOfDaysBeforeToday + 1, \"days\")\n    );\n}\n\nvalue = !!data.selectedOfferingEndDate && !isAfterGivenDaysBeforeNow(data.selectedOfferingEndDate,data.maximumAllowedDaysAfterStudyEndDate);",
```

Comment:
Can we add a comment explaining that the same logic exists in other form.io definitions and is also replicated on the server-side calculations based on const `COE_MAX_ALLOWED_DAYS_PAST_STUDY_PERIOD`.